tpm2/test/kdf_test.go

```diff
@@ -70,3 +70,87 @@ func TestKDFa(t *testing.T) {
 		}
 	}
 }
+
+func TestKDFe(t *testing.T) {
+	hashAlg := AlgSHA256
```

Comment:
Would is be possible to get 1 test case for sha1 and one for sha256, instead of 2 for sha256

Reply:
Added, as well as some more edge cases